feature_space/PPO.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+from ShmekelCore import Feature
+
+class PPO(Feature):
+    def __init__(self,fast_range=5,slow_range=10 , **kwargs):
+        """
+        use this method to define the parameters of the feature
+        """
+        super(PPO, self).__init__(**kwargs)
+
+        self.time_delay = slow_range
+        self.is_numerical = True
+
+    def getSma(self,icolumn=None,irange=None):
```

Comment:
you can merge your destination branch (feature/feature_space) to this branch.
then you can use the math.py module and call smooth moving average from there